packages/atomic-react/rollup.config.mjs

```diff
@@ -71,19 +77,48 @@ const plugins = [
   }),
 ];
 
+const pluginsCJS = [
+  json(),
+  nodePolyfills(),
+  typescript(),
+  commonjs(),
+  nodeResolve(),
+  replace({
+    delimiters: ['', ''],
```

Comment:
'','' ?

Reply:
Just copied the plugins from IIFE. I would like to keep this as it is and rework it in https://coveord.atlassian.net/browse/KIT-3541 .